### DISCUSSION TITLE
q-menu not visible in DOM when closed

### DISCUSSION THREAD

**Discussion by MaceOne** [quasarframework/quasar#16683] [General - Components / Directives / etc]
Hello, i made some shortcuts via document.getElementById which i would like to call inside the the q-menu but since my q-item with the id is not visible in DOM, it is not possible for me to get my element and click it. Or have i missed one of the q-menu props?  

I could use q-expansion-item, but it's pushing all my content down

**Comment by metalsadman** [comment]
provide a codepen/snippet.
